Update styles.css

M8inC opened the pull request:
Added a special cover height statement for Firefox. 

The Startpage and the posts headers are upset in FF - added a fix to adress this.
not-working-example: https://www.laviska.com/
working-example: https://www.ohnemotor.de 

;)

claviska commented:
Thanks for submitting this. I can't see any difference in the cover between the blog index or blog post pages (Firefox 64, macOS). What exactly does this fix?

M8inC commented:
That's strange. I put four screenshots in my dropbox, showing Firefox v64 (Windows, but Linux behaves equally) an chrome side by side. The cover image on your landigpage is "compressed" in FF, whilst it show correctly in chrome. On my page, everything is fine, after adding the patch to css.

https://www.dropbox.com/sh/08fe7go78qr4bbq/AAAWVyhQ0zrSPqWMnGDMy9YXa?dl=0

claviska commented:
Not sure why I wasn't seeing that the other day, but I see it today. It looks like this is caused by [`display: table` in style.css](https://github.com/Postleaf/empower-theme/blob/master/css/styles.css#L577). Evidently, Firefox doesn't honor `min-height` for tables.

The goal was to vertically center the header content, so a more consistent fix will be to use flexbox:

```css
.cover {
  display: flex;
  justify-content: center;
  align-items: center;
}
```

M8inC commented:
I'm not very educated in css, so I tried around a little to achive my goal. But, i'm of your opinion - your solution is more consistent. 

I removed [``display:table``](https://github.com/Postleaf/empower-theme/blob/7a198b7c00443f5388b3a5b7a10c43e60569cbbb/css/styles.css#L577) from style.css and added your fix - now https://www.ohnemotor.de works on firefox like a charme ;)

Thanks for your support - i will close the Pull-Request now.

claviska commented:
Fixed in 73e6bd5571168458e82b4b583c159852f3752332
